plugins/scaffolder-backend/src/scaffolder/stages/prepare/gitlab.test.ts

```diff
@@ -70,16 +70,17 @@ describe('GitLabPreparer', () => {
   const mockGitClient = {
     clone: jest.fn(),
   };
+  const logger = getVoidLogger();
 
   jest.spyOn(Git, 'fromAuth').mockReturnValue(mockGitClient as any);
 
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
-  ['gitlab', 'gitlab/api'].forEach(protocol => {
+  ['gitlab'].forEach(protocol => {
```

Comment:
so we remove that type as well, as mentioned above

also maybe skip the iteration now :)